composer.json

```diff
@@ -18,5 +18,11 @@
     "phpdocumentor/reflection-common": "^1.0@dev",
     "phpdocumentor/reflection-docblock": "^3.0@dev",
     "sebastian/diff": "^1.3"
+  },
+  "require-dev": {
+    "phpunit/phpunit": "^6.0"
+  },
+  "scripts": {
```

Comment:
Yes I prefer this way (it doesn't pollute the `composer.json` file with CI related commands).